src/VaultPHP/Response/MetaData.php

```diff
@@ -113,13 +113,30 @@ public function getAuth()
     }
 
     /**
-     * @return array|null
+     * @return array
```

Comment:
Hint: IDEs like PHPStorm (and also Psalm, I think) also understand array notations like `string[]`. This would save you the `@var string $something` annotations in the next methods.

```suggestion
     * @return string[]
```